packages/opentelemetry-api/src/trace/NoopTracer.ts

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-import { HttpTextFormat, Span, SpanOptions, Tracer } from '..';
-import { NOOP_HTTP_TEXT_FORMAT } from '../context/propagation/NoopHttpTextFormat';
+import { HttpTextPropagator, Span, SpanOptions, Tracer } from '..';
+import { NOOP_HTTP_TEXT_FORMAT } from '../context/propagation/NoopHttpTextPropagator';
```

Comment:
I think we should rename `NOOP_HTTP_TEXT_FORMAT` to `NOOP_HTTP_TEXT_PROPAGATOR`, WDYT?